evaluation/mint/scripts/run_infer.sh

```diff
@@ -15,8 +15,10 @@ echo "AGENT_VERSION: $AGENT_VERSION"
 export PYTHONPATH=$(pwd)
 
 COMMAND="poetry run python ./evaluation/mint/run_infer.py \
+    --llm-config $MODEL_CONFIG \
     --max-iterations 5 \
     --max-propose-solution 2 \
+    --eval-num-workers 1 \
```

Comment:
Is it one because it doesn't support parallel evaluation? In that case could you add a simple comment?

Reply:
Ah no, it can run in parallel actually, let me remove it!